internal/cortex/frontend/transport/utils/utils.go

```diff
@@ -159,7 +156,7 @@ func (f *FailedQueryCache) UpdateFailedQueryCache(err error, query url.Values, q
 	queryExpressionRangeLength := getQueryRangeSeconds(query)
 	// TODO(hc.zhu): add a flag for the threshold
 	// The current gateway timeout is 5 minutes, so we cache the failed query running longer than 5 minutes - 10 seconds.
-	if queryResponseTime > time.Second * (60 * 5 - 10) {
+	if queryResponseTime > time.Second*(60*5-10) {
```

Comment:
Can you make the magic number a flag since you are touching this part? :)

Reply:
I didnt, it was due to go format

Reply:
will do it in a separate PR